src/test/java/teammates/test/cases/BaseTestCaseWithBackDoorApiAccess.java

```diff
@@ -10,26 +10,55 @@
 import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
 import teammates.common.datatransfer.attributes.InstructorAttributes;
 import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
 import teammates.test.driver.BackDoor;
+import teammates.test.driver.retry.RetryManager;
+import teammates.test.driver.retry.RetryableTaskReturns;
 
 /**
  * Base class for all test cases which are allowed to access the Datastore via {@link BackDoor}.
  */
 public abstract class BaseTestCaseWithBackDoorApiAccess extends BaseTestCaseWithDatastoreAccess {
 
+    protected AccountAttributes getAccount(String googleId) {
```

Comment:
I see. Yes, the design of abstract class `BaseTestCaseWithDatastoreAccess.java` may not be that good. It require us to pass an object (a bit weird) to retrieve data. (e.g. `protected abstract AccountAttributes getAccount(AccountAttributes account);`) All the implementation actually only make use of some fields (in this case, only `googleId` is used).